extensions/amp-story/0.1/animation-presets.js

```diff
@@ -179,4 +179,56 @@ export const PRESETS = {
       return whooshIn(offsetX, 0, 0, 0);
     },
   },
+  'pan-left': {
+    duration: 1000,
+    easing: 'linear',
+    keyframes(dimensions) {
+      const offsetX = dimensions.pageWidth - dimensions.targetWidth;
+      const offsetY = dimensions.pageHeight - dimensions.targetHeight / 2;
+      return translate2d(offsetX, offsetY, 0, offsetY);
+    }
+  },
+  'pan-right': {
+    duration: 1000,
+    easing: 'linear',
+    keyframes(dimensions) {
+      const offsetX = dimensions.pageWidth - dimensions.targetWidth;
+      const offsetY = dimensions.pageHeight - dimensions.targetHeight / 2
+      return translate2d(0, offsetY, offsetX, offsetY);
+    }
+  },
+  'pan-down': {
+    duration: 1000,
+    easing: 'linear',
+    keyframes(dimensions) {
+      const offsetX = -dimensions.targetWidth / 2;
+      const offsetY = dimensions.pageHeight - dimensions.targetHeight;
+      return translate2d(offsetX, 0, offsetX, offsetY);
+    },
+  },
+  'pan-up': {
+    duration: 1000,
+    easing: 'linear',
+    keyframes(dimensions) {
+      const offsetX = -dimensions.targetWidth / 2;
+      const offsetY = dimensions.pageHeight - dimensions.targetHeight;
+      return translate2d(offsetX, offsetY, offsetX, 0);
+    }
+  },
+  'zoom-in': {
+    duration: 1000,
+    easing: 'linear',
+    keyframes: [
+      {transform: 'scale(1,1)'},
+      {transform: 'scale(3,3)'}
```

Comment:
As seen offline, we might want to experiment a bit with that and see if we're happy with the image quality :)

Reply:
Yep, will speak with Hong from UX today to see what she thinks 👍